scripts/setup_host.sh

```diff
@@ -215,6 +215,10 @@ function setup_sof() {
     $CIV_WORK_DIR/scripts/setup_audio_host.sh
 }
 
+function enable_headset() {
```

Comment:
can it be integrated into setup_sof()?

Reply:
I need to verify this in GVT-d also once. Please give me some time.